src/pages/basicAuthPage.ts

```diff
@@ -0,0 +1,28 @@
+import {expect, Page} from '@playwright/test';
+import PlaywrightWrapper from '../helper/wrapper/playwrightWrappers';
+import Assert from '../helper/wrapper/assert';
+
+export default class BasicAuthPage {
+  private base: PlaywrightWrapper;
+  private assert: Assert;
+
+  constructor(private page: Page) {
+    this.base = new PlaywrightWrapper(page);
+  }
+
+  public Elements = {
```

Comment:
class property names should follow camel case rule.

Reply:
Fixed